content/authors/jha-amit/_index.md

```diff
@@ -0,0 +1,53 @@
+# Display name
```

Comment:
I think the first line is missing from this file. It needs to start with three dashes on the first line (`---`).